tests/address/serializers/test_phone_number.py

```
@@ -0,0 +1,16 @@
+import pytest
+
+from address.serializers import AddressSerializer
+from address.exceptions import AddressException
+
+
+def test_invalid_phone_num():
+    serializer = AddressSerializer()
+    with pytest.raises(Exception):
```

Ruff (0.8.2)

`pytest.raises(Exception)` should be considered evil

(B017)